server/datastore/mysql/queries_test.go

```diff
@@ -98,6 +98,7 @@ func testQueriesApply(t *testing.T, ds *Datastore) {
 			Query:       "select baz from bar",
 			Logging:     fleet.LoggingSnapshot,
 			DiscardData: true,
+			LabelsIncludeAny:   []fleet.LabelIdent{{LabelID: fooLabel.ID, LabelName: fooLabel.Name}},
```

Comment:
This updated test fails on `main` (reproducing the issue), and passes on this branch.